File: python/1012.py
entrada = input().split(" ")

a, b, c = entrada

triangulo = (float(a) * float(c)) / 2
circulo = (3.14159 * float(c) * float(c))
trapezio = ((float(a) + float(b)) / 2) * float(c)
quadrado = float(b) * float(b)
retangulo = float(a) * float(b)

print("TRIANGULO: {:.3f}".format(triangulo))
print("CIRCULO: {:.3f}".format(circulo))
print("TRAPEZIO: {:.3f}".format(trapezio))
print("QUADRADO: {:.3f}".format(quadrado))
print("RETANGULO: {:.3f}".format(retangulo))

File: javascript/1012.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var valores = lines.shift().split(" ");

let A = parseFloat(valores.shift());
let B = parseFloat(valores.shift());
let C = parseFloat(valores.shift());
let circulo = 3.14159 * C * C;
let quadrado = B * B;
let retangulo = A * B;
let triangulo = (A * C) / 2;
let trapezio = ((A + B) / 2) * C;

console.log("TRIANGULO: " + triangulo.toFixed(3));
console.log("CIRCULO: " + circulo.toFixed(3));
console.log("TRAPEZIO: " + trapezio.toFixed(3));
console.log("QUADRADO: " + quadrado.toFixed(3));
console.log("RETANGULO: " + retangulo.toFixed(3));
